database/query/datasources.sql

```diff
@@ -0,0 +1,73 @@
+-- CreateDataSource creates a new datasource in a given project.
```

Comment:
Should we have delete on cascade based on the data source id for functions?

Reply:
We have that https://github.com/mindersec/minder/blob/main/database/migrations/000108_data_sources.up.sql#L47